VndbSharp/VndbUtils.cs

```diff
@@ -56,6 +56,14 @@ public static void WithHttpMessageHandler(Func<Task<HttpMessageHandler>> callbac
 			VndbUtils._httpClientHandlerCallback = callback;
 		}
 
+		/// <summary>
+		/// Validate Flags by Method
+		/// </summary>
+		/// <param name="method"></param>
+		/// <param name="flags"></param>
+		/// <param name="invalidFlags"></param>
+		/// <returns></returns>
```

Comment:
```suggestion
		/// <returns>True if there are no invalid flags, otherwise false</returns>
```

Is that right? Returns true if no invalid? Or is it returns false if there are no invalid?